images/stackstorm/bin/entrypoint-1ppc.sh

```diff
@@ -5,24 +5,30 @@ crudini --set ${ST2_CONF} auth    api_url     ${ST2_API_URL}
 crudini --set ${ST2_CONF} mistral api_url     ${ST2_API_URL}
 crudini --set ${ST2_CONF} mistral v2_base_url ${ST2_MISTRAL_API_URL}
 
+# Configure CORS to accept any source
+# st2api gunicorn process is directly exposed to clients in 1ppc mode
```

Comment:
Why are we exposing gunicorn to all clients if we're setting up nginx?

Reply:
@Mierdin nginx is only used in `st2web` container. So with other containers like st2auth or st2api, `gunicorn` process is directly exposed, not via nginx, as following the rule of *one process per container*

Reply:
Makes sense 👍